lib/3scale/backend/server/falcon.rb

```diff
@@ -4,7 +4,7 @@ module Server
       class Falcon
         extend ThreeScale::Backend::Server::Utils
 
-        CONFIG = 'falcon.rb'
+        CONFIG = 'falcon_config.rb'
```

Comment:
I think it was a good idea to rename, because there was two files in the codebase named `falcon.rb`, and it was confusing. It's better now.